tests/migration_test.go

```diff
@@ -2617,7 +2629,12 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 
 				dv := libdv.NewDataVolume(
 					libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskFedoraTestTooling)),
-					libdv.WithPVC(sc, cd.FedoraVolumeSize, k8sv1.ReadWriteMany, k8sv1.PersistentVolumeBlock),
+					libdv.WithPVC(
+						libdv.PVCWithStorageClass(sc),
```

Comment:
Would be great to have an option that would set block (volume mode) and relevant class at once. Just thinking out loud...

Reply:
The class is always a parameter. We can't guess it.

Reply:
What I mean is to create a "super" option... For example, when I want block volume I need 2-3 options(specify sc, mode, and size). These options could be represented by one option "DefaultBlockPVC" or so...

Reply:
This is good idea, but I think this better fits some kind of factory or test helper, and not this libdv builder.